lib/buyers/tests/test_api.py

```diff
@@ -63,18 +67,28 @@ def test_add_unauthenticated_buyer(self):
         eq_(buyer.authenticated, False)
         eq_(res.json['authenticated'], False)
 
+    def create(self, **kwargs):
+        defaults = {'uuid': self.uuid, 'pin': self.pin, 'email': self.email,
+                    'locale': 'en-US'}
+        defaults.update(kwargs)
+        return Buyer.objects.create(**defaults)
+
     def test_filter(self):
-        self.client.post(self.list_url, data={'uuid': self.uuid})
+        self.create()
         res = self.client.get(self.list_url + '?uuid=%s' % self.uuid)
         eq_(res.status_code, 200)
         eq_(res.json['meta']['total_count'], 1)
         eq_(res.json['objects'][0]['uuid'], self.uuid)
 
-    def create(self, **kwargs):
-        defaults = {'uuid': self.uuid, 'pin': self.pin, 'email': self.email,
-                    'locale': 'en-US'}
-        defaults.update(kwargs)
-        return Buyer.objects.create(**defaults)
+    def test_filter_email(self):
+        self.create()
+        res = self.client.get(self.list_url, {'email': self.email}).json
+        eq_(res['objects'][0]['email'], self.email)
+
+    def test_multiple_email(self):
+        self.create()
+        with self.assertRaises(ValueError):
+            self.client.get(self.list_url, {'email': [self.email, self.email]})
```

Comment:
shouldn't this return a 400? Or is the ValueError coming from django test tools?